lib/Common/Memory/RecyclerPageAllocator.h

```diff
@@ -41,8 +41,11 @@ class RecyclerPageAllocator : public IdleDecommitPageAllocator
 #endif
 
 private:
+
 #if ENABLE_BACKGROUND_PAGE_ZEROING
     ZeroPageQueue zeroPageQueue;
+#elif ENABLE_BACKGROUND_PAGE_FREEING
```

Comment:
back ground zeroing has dependency on back ground freeing.

---
In reply to: [89222601](https://github.com/Microsoft/ChakraCore/pull/2071#discussion_r89222601) [](ancestors = 89222601)